src/claimtrie.cpp

```diff
@@ -2491,12 +2346,12 @@ bool CClaimTrieCache::incrementBlock(insertUndoType& insertUndo, claimQueueRowTy
                     {
                         LogPrintf("%s(): An inconsistency was found in the claim queue. Please report this to the developers:\nClaim found in name queue but not in height based queue:\nname: %s, txid: %s, nOut: %d, nValidAtHeight in name based queue: %d, current height: %d\n", __func__, nameToCheck, itQueueName->outPoint.hash.GetHex(), itQueueName->outPoint.n, itQueueName->nHeight, nCurrentHeight);
                     }
-                    assert(found);
+                    //assert(found);
```

Comment:
If this assert fails it looks like it compares noramalized and not normalized name, that's pretty likely when it used in increment block.